### PR TITLE
Fix/temporal scopes

### LINKED ISSUE #94: Fixity/Ingest/Digest created_after and created_before are reversed
Pulled from `app/models/fixity_check.rb`

``` ruby
  scope :created_after, ->(time) { where("created_at < ?", time) unless time.blank? }
  scope :created_before, ->(time) { where("created_at > ?", time) unless time.blank? }
```

We can see `:created_after` will return items before `time` and `:created_before` will do the opposite.

We can compare to `app/models/bag.rb` as a sanity check as well (altered to match the ordering above)

``` ruby
  scope :updated_after, ->(time) { where("updated_at > ?", time) unless time.blank? }
  scope :updated_before, ->(time) { where("updated_at < ?", time) unless time.blank? }
```

### PR DESCRIPTION
**Pull request by malakai97**
Resolves #94

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -56,6 +56,8 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.alias_it_should_behave_like_to :it_has_behavior, 'has behavior:'
+
```
Comment (dazza-codes) [inline spec/spec_helper.rb]: The conventional invocation for shared examples is simply `it_behaves_like`.  Why do we need something unconventional?

File: spec/fabricators/fixity_check_fabricator.rb [hunk 2]
```diff
@@ -9,6 +9,6 @@
   bag
   node
   success true
-  fixity_at 3.seconds.ago
   created_at 1.second.ago
+  fixity_at {|attrs| (attrs[:created_at] || 1.second.ago) - 3.seconds}
 end
```
Comment (dazza-codes) [inline spec/fabricators/fixity_check_fabricator.rb:16]: As a novice reading this piece of code, it's not intuitive; please add commentary to explain the purpose of the temporal fields and the intended relationship between their values.  Add references to model validation, as necessary.

**Review by dazza-codes — CHANGES_REQUESTED** [review]
Generally OK, although prefer to see conventional invocation for shared examples.  Also, not entirely clear why this change adds new scopes on models that did not have them before - are these scopes actually used anywhere in controllers or views?

File: spec/spec_helper.rb (hunk 1 above)
Comment (malakai97) [reply spec/spec_helper.rb]: done

File: spec/fabricators/fixity_check_fabricator.rb (hunk 2 above)
Comment (malakai97) [reply spec/fabricators/fixity_check_fabricator.rb:16]: addressed w/ comment

**Comment by malakai97** [comment]
> ...not entirely clear why this change adds new scopes on models that did not have them before...

Added scopes to models based on the specification.  Did not want to touch controllers in this PR to complete the enabling of the spec-defined behavior.

**Review by dazza-codes — APPROVED** [review]
Looks good to go with these changes.

**Pull request merged**